hexa/pipelines/models.py

```diff
@@ -195,6 +200,14 @@ class Meta:
     memory_limit = models.CharField(blank=True, max_length=32)
     recipients = models.ManyToManyField(User, through="PipelineRecipient")
 
+    type = models.CharField(
+        max_length=200,
+        blank=False,
+        choices=PipelineType.choices,
+        default=PipelineType.DEFAULT,
+    )
+    notebook = models.CharField(max_length=200, null=True, blank=True)
```

Comment:
`notebookPath` seems like a better idea. 200 chars seems too narrow for this. It's better to use `TextField`